docs/common/faq.md

```diff
@@ -3,8 +3,10 @@
 [Go back to Documentation home page ](../README.md)
 
 ### Do the Ansible Modules support Active Directory based Cohesity Accounts?
-Yes.  You can configure your domain specific credentials in one of two formats.
+Yes.  You can configure your domain specific credentials in following formats.
 - username@domain
+- domain/username@tenant # AD Tenant User
+- LOCAL/username@tenant  # Local Tenant User
```

Comment:
give an example like AD.domain.com/username@tenant

Reply:
Does this looks good?
username@`AD.domain.com`
`AD.domain.com`/username@tenant # AD Tenant User
LOCAL/username@tenant